L1Trigger/L1THGCal/python/hgcalBackEndLayer2Producer_cfi.py

```diff
@@ -149,10 +150,17 @@
                                      dr_bylayer = cms.vdouble([0.015]*15)
                                      )
 
-phase2_hgcalV10.toModify(energy_interpretations_em,
-                         layer_containment_corrs=cms.vdouble(0., 0., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1.),
-                         scale_correction_coeff=cms.vdouble(0., 0.),
-                         )
+phase2_hgcalV10.toModify(
+        energy_interpretations_em,
+        layer_containment_corrs=cms.vdouble(0., 0.0, 2.0, 2.0, 0.63, 0.5, 1.91, 0.5, 1.3, 1.27, 0.66, 0.76, 0.78, 2.0, 2.0),
+        scale_correction_coeff=cms.vdouble(-43.074539, 76.823082),
```

Comment:
- The layer coefficients are varying a lot compared to the values for V9, with very low values as well as larger values.
- Also, the eta corrections coefficients are much larger than before
Are these new coeffs been derived in a similar way as before?

Reply:
At least conceptually:
- minimization of the RMS of E residuals w.r.t GEN on photons with PU0 (2<pt<100GeV)
- linear fit of the E residuals vs eta on electrons @ PU200

I didn't, however, calibrate myself the v9 and those values have never been used (not even in the TDR) so I am less confident of those numbers. Note that the signs of the eta correction seem to be completely off (see plots below)

The pt response vs eta before the calibration looks like:

![canvas(96)](https://user-images.githubusercontent.com/4802196/83022212-77131a80-a02b-11ea-88e5-146f8431509b.png)

while after the calibration becomes:
![canvas(95)](https://user-images.githubusercontent.com/4802196/83022287-93af5280-a02b-11ea-863a-fb6cd2999d55.png)

The sign of the new corrections goes in the right direction. Still, the linear assumption seems to be sub-optimal at low eta.

As we discussed a while ago I have big doubts about the calibration procedure but, when looking at the "integral" across the whole eta, it does a decent job.

![canvas(94)](https://user-images.githubusercontent.com/4802196/83022725-3e277580-a02c-11ea-8ea6-536c1f51bc3d.png)

![canvas(97)](https://user-images.githubusercontent.com/4802196/83022953-93638700-a02c-11ea-9545-ca76246b1836.png)

I would be very happy if you could reproduce these results and compare performance (we should agree on the details like which estimator, which eta and pt range etc)